py3status/modules/rate_counter.py

```diff
@@ -1,168 +1,230 @@
 # -*- coding: utf-8 -*-
 """
-Display time spent and calculate the price of your service.
+Display calculated time spent and costs.
 
 Configuration parameters:
-    cache_timeout: how often to update in seconds (default 5)
-    config_file: file path to store the time already spent
-        and restore it the next session
-        (default '~/.i3/py3status/counter-config.save')
-    format: output format string
-        (default 'Time: {days} day {hours}:{mins:02d} Cost: {total}')
-    format_money: output format string
-        (default '{price}$')
-    hour_price: your price per hour (default 30)
-    tax: tax value (1.02 = 2%) (default 1.02)
+    button_reset: mouse button to reset the timer (default 3)
+    button_toggle: mouse button to toggle the timer (default 1)
+    config_file: specify a file to save time between sessions
+        (default '~/.config/py3status/rate_counter.save')
+    format: display format for this module
+        *(default 'Rate Counter [\?color=state [\?not_zero {days} days ]'
+        '[\?not_zero {hours}:]{minutes}:{seconds}'
+        '[\?color=darkgray&show \|]${total}]')*
+    rate: specify the hourly pay rate to use (default 30)
+    tax: specify the tax value to use, 1.02 is 2% (default 1.0)
```

Comment:
changing default tax rate, you'll screw users

Reply:
It does not make sense to default tax rate for users all over the world. `1.0` should leave things as-is (without specified tax rates).

Reply:
that's your opinion, not the author's